src/syscheckd/registry/registry.c

```diff
@@ -482,7 +506,8 @@ fim_registry_key *fim_registry_get_key_data(HKEY key_handle, const char *path, c
         key->mtime = get_registry_mtime(key_handle);
     }
 
-    key->last_event = time(NULL);
+    //key->last_event = time(NULL);
```

Comment:
This statement could be deleted

Reply:
I changed those statements as a reminder to change the current timestamp stored in last_event for a correct generation of events for when dbsync updated the issue with deltas.